bugsnag-android-core/src/main/java/com/bugsnag/android/internal/JsonHelper.kt

```diff
@@ -31,6 +34,20 @@ internal object JsonHelper {
         }
     }
 
+    fun serialize(streamable: JsonStream.Streamable): ByteArray {
+        return ByteArrayOutputStream().use { baos ->
+            JsonStream(PrintWriter(baos).buffered()).use(streamable::toStream)
```

Comment:
`buffered()` doesn't contribute anything here since we're piping everything into a buffer
```suggestion
            JsonStream(PrintWriter(baos)).use(streamable::toStream)
```